lib/puppet/provider/ssh_authorized_key/parsed.rb

```diff
@@ -64,6 +64,7 @@ def trusted_path
       path = path.dirname
     end
     Puppet.debug('Path trusted, writing the file as the current user')
+    return true
```

Comment:
Thanks! There should be a test to verify this behavior. Do you know if one exists already?